### ISSUE TITLE
[BUG] Video pause after app go to background

### ISSUE THREAD

**Issue by qhu91it** [jhomlala/betterplayer#268]
**Describe the bug**
I setting play on the background and enable showNotification on data source, when app go to background the video play auto-pause.

```
BetterPlayerDataSource betterPlayerDataSource = BetterPlayerDataSource(
      BetterPlayerDataSourceType.network,
      videoUrl,
      cacheConfiguration: BetterPlayerCacheConfiguration(
        useCache: true,
      ),
      notificationConfiguration: BetterPlayerNotificationConfiguration(
        showNotification: true,
        title: video.snippet.title,
        author: snippet.channelTitle,
        imageUrl: video.snippet.thumbnails.medium.url,
      ),
    );
    betterPlayerController = BetterPlayerController(
      BetterPlayerConfiguration(
        autoPlay: true,
        autoDispose: true,
        controlsConfiguration: BetterPlayerControlsConfiguration(
          playerTheme: BetterPlayerTheme.custom,
        ),
      ),
      betterPlayerDataSource: betterPlayerDataSource,
    );
```

I already catch event app pause or inactive to call start but no help

`betterPlayerController.videoPlayerController.play();`

**To Reproduce**
1. Enable UIBackgroundModes audio on ios project
2. Enable and setting BetterPlayerNotificationConfiguration
3. Play the video
4. Go to os home screen (app goes to background)

**Expected behavior**
The media play continue playing when the app goes to the background

**Screenshots**

**Flutter doctor**
```
[✓] Flutter (Channel stable, 1.22.6, on macOS 11.2 20D64 darwin-x64, locale en)
    • Flutter version 1.22.6 at /Users/mac/fvm/versions/stable
    • Framework revision 9b2d32b605 (13 days ago), 2021-01-22 14:36:39 -0800
    • Engine revision 2f0af37152
    • Dart version 2.10.5


[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/mac/Library/Android/sdk
    • Platform android-30, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.2, Build version 12B45b
    • CocoaPods version 1.10.1

[!] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.53.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.19.0

[✓] Connected device (2 available)

```
**Better Player version**
- Version: 0.0.49

**Smartphone (please complete the following information):**
 - Device: iPhoneX
 - OS: iOS14.3

**Additional context**

**Comment by jhomlala** [comment]
Fixed in 0.0.50.

**Comment by JenishMS** [comment]
still this issue there @jhomlala
